src/elements/content-preview/__tests__/PreviewNavigation.test.js

```diff
@@ -74,8 +81,20 @@ describe('elements/content-preview/PreviewNavigation', () => {
             expect(wrapper.find('PlainButton')).toHaveLength(1);
             wrapper.find('PlainButton').simulate('click');
 
-            expect(historyMock.push).toBeCalledTimes(1);
-            expect(historyMock.push).toBeCalledWith('/activity');
+            expect(historyMockDefault.push).toBeCalledTimes(1);
+            expect(historyMockDefault.push).toBeCalledWith('/activity');
+            expect(onNavigateRightMock).toHaveBeenCalled();
+        });
+
+        test('should render navigation correctly from metadata deeplinked URL ', () => {
```

Comment:
Do we need a test for the left nav as well

Reply:
added 👍